component-lib/src/business/Table/Table.tsx

```diff
@@ -16,30 +14,48 @@ export const TableBodyCell: React.FC<TableBodyCellProps> = (props) => {
     const { children } = props;
 
     return (
-        <td onClick={props.onClick} className={cs({
-            "data-table__cell": true,
-            "data-table__cell--right-aligned": props.rightAligned
-        })}>{children}</td>
+        <td onClick={props.onClick}
```

Comment:
We should also stop propagation on click to prevet a row click to fire when we have an onClick on the cell 
```suggestion
        <td  onClick={e => {
        if (props.onClick) {
          e.stopPropagation();
          props.onClick();
        }
      }}
```